packages/liveblocks-zustand/src/index.ts

```diff
@@ -41,12 +41,12 @@ export type LiveblocksContext<
    * Enters a room and starts sync it with zustand state
    * @param roomId The id of the room
    */
-  readonly enterRoom: (roomId: string) => void;
+  readonly enterRoom: (roomId: string) => () => void;
   /**
-   * Leaves a room and stops sync it with zustand state.
-   * @param roomId The id of the room
+   * Leaves the currently entered room and stops sync it with zustand state, if
+   * any. If enterRoom was not called before, this is a no-op.
    */
-  readonly leaveRoom: (roomId: string) => void;
+  readonly leaveRoom: () => void;
```

Comment:
Is it a breaking change?

Reply:
Same here. Removing the last param from a function is not a breaking change, semver-wise.

Reply:
As agreed after a Slack discussion, I changed the type definition to deprecate these overloads instead of removing the params, so technically it's no longer a breaking change: https://github.com/liveblocks/liveblocks/commit/0f3d3bb27ea025660365e386d09df28527872141

However, I'm honestly unsure if this leads to a better DX this way 🤔 I find the IDE hints not _that_ clear, and you have to dive into the type definitions to get the necessary information, and honestly the types aren't that clear. Maybe #staypragmatic here and not care too deeply in this case? Seeing a single/simple function definition without a room ID seems much clearer to see that it's no longer needed.

@GuillaumeSalles What do you think?

Reply:
Makes sense, let's just put it in the release notes